client/src/index.css

```diff
@@ -43,3 +43,16 @@ p {
   --highlight-color: #e882a8;
   --arrow-color: #ffbf3e;
 }
+
+@media only screen and (min-width: 992px) {
+  
+  html {
+    background-color: rgb(247, 241, 241);
+  }
+  
+  body {
+    width: 60%;
```

Comment:
Hi Ali, instead of setting the width of the body to 60% , which makes a whole outline around the display:

![Screen Shot 2019-04-27 at 18 19 59](https://user-images.githubusercontent.com/37771591/56852929-20fc1100-6919-11e9-8b27-3153384d9035.png)

Leave the body at 100% and center things that aren't already center. For example the banner:
![Screen Shot 2019-04-27 at 18 21 46](https://user-images.githubusercontent.com/37771591/56852945-5ef93500-6919-11e9-8617-b4bda7977a0b.png)

Can be fixed by adding a margin:0 auto;
![Screen Shot 2019-04-27 at 18 22 46](https://user-images.githubusercontent.com/37771591/56852954-851ed500-6919-11e9-9e3d-e25be004930d.png)